Doc/library/pathlib.rst

```diff
@@ -1026,6 +1019,10 @@ call fails (for example because the path doesn't exist).
    wildcards. Set *follow_symlinks* to ``True`` to always follow symlinks, or
    ``False`` to treat all symlinks as files.
 
+   .. tip::
+      Set *follow_symlinks* to ``True`` or ``False`` to improve performance
+      of recursive globbing.
+
    .. audit-event:: pathlib.Path.glob self,pattern pathlib.Path.glob
 
    .. versionchanged:: 3.11
```

Comment:
Should I move the `.. versionchanged` block for trailing slash support into the "pattern language" section?

Reply:
Answering my own question: yes I think so!